docs/notebooks/01_infrastructure_setup.ipynb

```diff
@@ -0,0 +1,280 @@
+   "execution_count": null,
+   "metadata": {
+    "id": "ur8xi4C7S06n"
+   },
+   "outputs": [],
+   "source": [
+    "# Copyright 2023 Google LLC\n",
+    "#\n",
+    "# Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "# you may not use this file except in compliance with the License.\n",
+    "# You may obtain a copy of the License at\n",
+    "#\n",
+    "#     https://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "# Unless required by applicable law or agreed to in writing, software\n",
+    "# distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "# See the License for the specific language governing permissions and\n",
+    "# limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "JAPoU8Sm5E6e",
+    "tags": []
+   },
+   "source": [
+    "# Getting Started with Vertex AI Turbo Templates\n",
+    "\n",
+    "This notebook sets up infrastructure to run production-ready pipelines on Google Cloud. Follow this three-part notebook series to get started in a local Jupyter notebook or in [Vertex AI Workbench](https://cloud.google.com/vertex-ai-notebooks):\n",
+    "\n",
+    "1. **[Infrastructure Setup](./02_run_pipelines.ipynb) - this notebook**\n",
+    "1. [Run Pipelines](./02_run_pipelines.ipynb)\n",
+    "1. [Infrastructure Clean Up](./02_run_pipelines.ipynb)\n",
+    "\n",
+    "\n",
+    "**Prerequisites:**\n",
+    "\n",
+    "- [Google Cloud SDK (gcloud)](https://cloud.google.com/sdk/docs/quickstart)\n",
+    "- Make\n",
+    "- [Terraform](https://www.terraform.io)\n",
+    "\n",
+    "**For Vertex AI Workbench users**: Uncomment and execute the following cell to install Terraform."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# ! bash ./scripts/install_terraform.sh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Authenticate\n",
+    "\n",
+    "#### Set your project ID\n",
+    "\n",
+    "**If you don't know your project ID**, try the following:\n",
+    "* Run `gcloud config list`.\n",
+    "* Run `gcloud projects list`.\n",
+    "* See the support page: [Locate the project ID](https://support.google.com/googleapi/answer/7014113)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "VERTEX_PROJECT_ID = \"my-project-id\"\n",
+    "! gcloud auth login\n",
+    "! gcloud config set project {VERTEX_PROJECT_ID}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clone Code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! git clone -b develop https://github.com/teamdatatonic/vertex-pipelines-end-to-end-samples"
```

Comment:
Not sure about cloning the repo in the notebook here. The terraform install cell above assumes the user is running the notebook inside the repo already.

I think it makes sense to assume a user has cloned the repo already and adjust the paths to be relative to the notebook(s)